src/libs/actions/Session/index.js

```diff
@@ -266,29 +261,13 @@ function signInWithShortLivedAuthToken(email, authToken) {
                 isLoading: false,
             },
         },
-        {
-            onyxMethod: CONST.ONYX.METHOD.MERGE,
-            key: ONYXKEYS.IS_TOKEN_VALID,
-            value: false,
-        },
     ];
 
     // If the user is signing in with a different account from the current app, should not pass the auto-generated login as it may be tied to the old account.
     // scene 1: the user is transitioning to newDot from a different account on oldDot.
     // scene 2: the user is transitioning to desktop app from a different account on web app.
     const oldPartnerUserID = credentials.login === email ? credentials.autoGeneratedLogin : '';
-    // eslint-disable-next-line rulesdir/no-api-side-effects-method
-    API.makeRequestWithSideEffects(
-        'SignInWithShortLivedAuthToken',
-        {authToken, oldPartnerUserID},
-        {optimisticData, successData, failureData},
-        null,
-    ).then((response) => {
-        if (response) {
-            return;
-        }
-        Navigation.navigate();
-    });
+    API.read('SignInWithShortLivedAuthToken', {authToken, oldPartnerUserID, shouldRetry: false}, {optimisticData, successData, failureData});
```

Comment:
I don't think we need the `shouldRetry` here do we? Read requests are not retryable by default IIRC and we have an open issue to remove this param entirely.